### DISCUSSION TITLE
Can you share the project file of the sample content you used

### DISCUSSION THREAD

**Discussion by LSLYWC** [yasirkula/UnityDynamicPanels#9] [Q&A]
Can you share the project file of the sample content you used，i need this，thanks。
![A0975BBA-FA2F-4027-A4A8-4092FE8321A0](https://user-images.githubusercontent.com/31689242/110288499-1174a580-8023-11eb-9991-9f86a0b75afa.png)

**Comment by yasirkula** [comment]
There you go: https://github.com/yasirkula/UnityDynamicPanels/issues/5

**Reply by LSLYWC** [reply]
Thank you very much！

**Comment by LSLYWC** [comment]
I imported my scene, but there was an error in the project, the error is：Assets\Plugins\RuntimeInspector\Scripts\RuntimeInspector\Fields\NumberRangeField.cs(25,30): error CS0115: 'NumberRangeField.CanBindTo(Type, MemberInfo)': no suitable method found to override
I'm using is unity2019.4.21f1c1. The package appears to be old......

**Reply by yasirkula** [reply]
Can you delete RuntimeInspector and reimport the latest version?

**Reply by LSLYWC** [reply]
The same result seems to be related to a. Net function. Which version of unity do you use?

**Reply by yasirkula** [reply]
I thought we had to import the other plugins separately, my bad. Now I saw that the plugins are already included in *Unityception.unitypackage*. You should delete RuntimeInspector, RuntimeSceneGizmo, DynamicPanels and IngameDebugConsole folders and then import Unityception.unitypackage.

I've [now updated](https://drive.google.com/open?id=1bAHEDF1AQ03rECr2kh9AxymClXrvrhLW) the unitypackage so that it uses the latest versions of all these plugins.
